templates/home.html

```diff
@@ -29,7 +29,7 @@
 </ul>
 {% if user.is_authenticated %}
 <div class="container text-center">
-    <h1 class="mt-5">Hi {{user.username }}!</h1>
+    <h1 class="mt-5">Hi {{user.username | upper}}!</h1>
```

Comment:
How we know `upper` tag being used here is built-in or custom one?

Reply:
I've changed upper to upper_value in my custom define functions as upper is also a built-in tag in Django.